api/v1alpha1/worker_types.go

```diff
@@ -301,7 +321,9 @@ type RolloutStep struct {
 	// routed to the new worker deployment version while this step is active.
 	//
 	// Acceptable range is [0,100].
-	RampPercentage float32 `json:"rampPercentage"`
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=100
+	RampPercentage int32 `json:"rampPercentage"`
```

Comment:
Same question as above -- can we support ramp values <1%?

Reply:
the server can handle it, yes. _should_ someone set a ramp to 0 or 100 is a different question (I think it's confusing, but we allowed it so..)